internal/cli/login.go

```diff
@@ -528,31 +531,43 @@ func (o *LoginOptions) Run(ctx context.Context, args []string) error {
 		return err
 	}
 
-	// Auto-select organization if enabled and user has access to only one
-	if response, err := c.ListOrganizationsWithResponse(ctx, &v1beta1.ListOrganizationsParams{}); err == nil && response.StatusCode() == http.StatusOK && response.JSON200 != nil && len(response.JSON200.Items) == 1 {
-		org := response.JSON200.Items[0]
-		if org.Metadata.Name != nil {
-			orgName := *org.Metadata.Name
-			o.clientConfig.Organization = orgName
-
-			displayName := ""
-			if org.Spec != nil && org.Spec.DisplayName != nil {
-				displayName = *org.Spec.DisplayName
-			}
-
-			if displayName != "" {
-				fmt.Printf("Auto-selected organization: %s %s\n", orgName, displayName)
-			} else {
-				fmt.Printf("Auto-selected organization: %s\n", orgName)
-			}
-		}
-	}
-
 	if err := o.clientConfig.Persist(o.ConfigFilePath); err != nil {
 		return fmt.Errorf("persisting client config: %w", err)
 	}
 
 	fmt.Println("Login successful.")
+
+	// Handle organization selection if enabled
+	if o.authConfig.OrganizationsEnabled != nil && *o.authConfig.OrganizationsEnabled {
+		if response, err := c.ListOrganizationsWithResponse(ctx, &v1beta1.ListOrganizationsParams{}); err == nil && response.StatusCode() == http.StatusOK && response.JSON200 != nil {
```

Comment:
If Organizations are enabled and the request failed, should we still be swallowing this err? Or would it be helpful to indicate that something went wrong with this call?

Reply:
Admittedly I am not a power user of organizations, but I think _I_ would prefer some indication. 

If 99.9% of the time I login and my org is set for me, but that .01% of the time it's not, I may have questions.

With Orgs enabled, the other flows - single org discovered, multiple orgs discovered, failing to persist config - result in additional information output.